camelAssemblies/src/main/resources/routeOverrides/fhir_sink/v3_21_0/fhir_sink.routes.yaml

```diff
@@ -0,0 +1,43 @@
+-   route-template:
+        id: Route templates from fhir_sink:v3_21_0 (modified)
+        from:
+            uri: vantiq://server.config?consumerOutputJsonStream=true&structuredMessageHeader=true
+            steps:
+            -   choice:
+                    precondition: true
+                    when:
+                    -   simple: ${properties:encoding} =~ 'JSON'
+                        steps:
+                        -   unmarshal:
+                                fhirJson:
+                                    fhir-version: '{{fhirVersion}}'
+                                    pretty-print: '{{prettyPrint}}'
+                    -   simple: ${properties:encoding} =~ 'XML'
+                        steps:
+                        -   unmarshal:
+                                fhirXml:
+                                    fhir-version: '{{fhirVersion}}'
+                                    pretty-print: '{{prettyPrint}}'
+            -   to:
+                    uri: fhir://{{apiName}}/{{methodName}}
+                    parameters:
```

Comment:
`inBody: "resource"` was in this list, but the underlying component doesn't seem to support it (blows up).  So we'll remove it.